share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,38 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ""); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
+            return Spice.failed('spotify');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "spotify",
+
+            // Customize these properties
+            name: "Spotify",
+            data: api_result.tracks.items,
+            meta: {
+                sourceName: "Spotify.com",
+                sourceUrl: 'https://api.spotify.com/v1/search?q=' + query.trim() + '&type=track',
```

Comment:
The "More at ..." currently links to the JSON response which users might not want to see. Is it possible to link to a query to https://play.spotify.com/?